pkgs/by-name/va/vanguards/package.nix

```
@@ -0,0 +1,35 @@
+{
+  python312Packages,
+  fetchFromGitHub,
+  lib,
+}:
+python312Packages.buildPythonApplication rec {
+  pname = "vanguards";
+  version = "0.3.1";
```

```suggestion
  version = "0.3.1-unstable-2023-10-31";
```
Since you use the latest commit and not the release